docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+# Databricks development workflow
+
+This guide demonstrates a development workflow for Kedro projects on Databricks using Databricks Repos. This hybrid workflow involves developing your Kedro project using a local Integrated Development Environment (IDE) and Databricks notebooks for testing purposes. This guide demonstrates this workflow using an example based on the Kedro PySpark Iris starter.
+This workflow has several advantages, enabling:
+
+- Use of powerful development features offered by an IDE not available on Databricks notebooks.
+- Version control of your project.
+- Multiuser development.
+
+An IDE offers benefits for development:
+
+- Auto-completion and suggestions for code, improving your development speed and accuracy.
+- Linters like Pylint or Flake8 can be integrated to catch potential issues in your code.
+- Static type checkers like Mypy can check types in your code, helping to identify potential type-related issues early in the development process.
+
+To set up these features, look for instructions specific to your IDE (for instance, [VS Code](https://code.visualstudio.com/docs/python/linting)).
+
+## Prerequisites
+
+- New or existing [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) with administrative privileges.
```

Comment:
Administrative privileges for what? It seems unlikely that our users (especially QB on client side) would have full admin rights for an AWS account. What's the specific rights they need to have to follow this flow?

Reply:
From the rest of the tutorial it's not clear to me why you need admin privileges? There's no mention of specific configurations or permissions.

Reply:
Btw, I actually tried this out on the Azure Databricks and it worked fine, so perhaps it's not necessary to specifically call out AWS things here?

Reply:
You're right about this, this was a leftover from our old documentation. I removed this in a commit that I hadn't pushed before your review.